import/n2mg_separate_files_cypherls.sh

```diff
@@ -52,6 +52,9 @@ sed -e 's/CREATE RANGE INDEX FOR (n:/CREATE INDEX ON :/g;' \
     -e '/^CREATE CONSTRAINT/d' $INPUT_SCHEMA >> "$OUTPUT_SCHEMA"
 
 cat "$INPUT_NODES" > "$OUTPUT_NODES"
+# Add backticks around property keys in any {key: value} patterns.
+sed -i -E -e 's/(\{|,|\s)([a-zA-Z_][a-zA-Z0-9_]*)(\s*:\s*)/\1`\2`\3/g' -e 's/`n`/n/g' "$OUTPUT_NODES"
```

Comment:
```suggestion
sed -i '' -E \
  -e 's/(\{|\s|,)([[:space:]]*)([a-zA-Z_][a-zA-Z0-9_]*)(\s*:\s*)/\1\2`\3`\4/g' \
  -e 's/`n`/n/g' "$OUTPUT_NODES"
```

Reply:
Hi @katarinasupe . Thank you for your suggestion! 
I modified your suggested changes a bit to make it work on Linux (Ubuntu). Since I was getting the following error on Ubuntu, it seems that Linux can't handle empty -i flag well:
```
sed: can't read : No such file or directory
```

Please let me know if the new command works on MasOS.

Reply:
Your command seems to be working on MacOS as well